collect_app/src/main/res/values/strings.xml

```diff
@@ -743,4 +743,5 @@
     <string name="preference_not_available">Not available</string>
     <string name="legacy_custom_server_paths">Legacy custom server paths</string>
     <string name="custom_server_paths">Custom server paths</string>
```

Comment:
Maybe "legacy" doesn't add anything and we should use "Custom server paths" everywhere. The detail text about removal is clear, I think.

Reply:
Makes sense. I'll change it!